pkgs/development/python-modules/ibis-framework/default.nix

```diff
@@ -78,6 +81,7 @@ buildPythonPackage rec {
     tables
     toolz
   ] ++ lib.optionals (pythonOlder "3.8") [
+    # TODO: remove when ibis 3.0.0 is released
```

Comment:
you could use version comparison for this. That way it will be automatically excluded after 3.0.0 and whenever we notice it we can clean it up

Reply:
Done. Thanks!